src/cls/rgw/cls_rgw_types.h

```diff
@@ -11,6 +11,8 @@
 #define CEPH_RGW_TAG_TIMEOUT 120
 #define CEPH_RGW_DIR_SUGGEST_LOG_OP  0x80
 #define CEPH_RGW_DIR_SUGGEST_OP_MASK 0x7f
+#define MP_META_SUFFIX ".meta"
+#define MP_MULTI_SUFFIX "_multipart"
```

Comment:
This isn't really a suffix is it?  This looks like it's supposed to match "RGW_OBJ_NS_MULTIPART" - however that omits the '_' which is inconvenient.